site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,173 @@
+
+With the previous VMM rewrite reducing memory use, the dependencies updated, and all the groundwork done, [Blinkhawk](https://github.com/FernandoS27) 
+pressed the metaphorical nuclear launch button and [released Project Prometheus](https://github.com/yuzu-emu/yuzu/pull/3955).
+yuzu now use up to 6 to 7 CPU threads (in ideal conditions) 
+compared to the previous 2 to 3. You should expect a performance boost in a lot of games, but still see some titles perform mostly the same due to being coded to only use a single thread.
+
+Now, some clarifications are needed for this change. Multicore support can’t be merged into our [Mainline](https://github.com/yuzu-emu/yuzu-mainline) release for now due to incompatibilities between Multicore and the [Master](https://github.com/yuzu-emu/yuzu) branch of yuzu. Work is being done to resolve the conflicts, but please have patience.
+Additionally, users with 2 cores, and either 2 or 4 threads, should not enable multicore as it will most likely result in a performance 
+loss for them due to the lack of physical cores on their CPUs. Our [hardware recommendations](https://yuzu-emu.org/help/quickstart/#hardware) 
+have been updated accordingly.
+
+## Unreal Engine fixes
+
+[Rodrigo](https://github.com/ReinUsesLisp) implemented [rendering more than one slice of 3D textures](https://github.com/yuzu-emu/yuzu/pull/4027), fixing the most glaring 
+issue in Unreal Engine 4 games, known as “the Rainbow”. This also improves the previous implementation that was used in 
+Xenoblade games.
+
+{{< single-title-imgs
+    "Your Excellency (OCTOPATH TRAVELER)"
+    "./01.png"
+    "./02.png"
+  >}}
+
+## Animal Crossing: New Horizons changes
+
+[Rodrigo](https://github.com/ReinUsesLisp) fixed `Animal Crossing: New Horizons` terrain borders in Vulkan by implementing 
+[constant attributes](https://github.com/yuzu-emu/yuzu/pull/3930). This is not a native extension, constant attributes have to be emulated in Vulkan as there is currently no official support for it.
+
+{{< single-title-imgs
+    "Beautiful beaches, now in Vulkan too (Animal Crossing: New Horizons)"
+    "./04.png"
+    "./05.png"
+  >}}
+
+[bunnei](https://github.com/bunnei) implemented [time zone support](https://github.com/yuzu-emu/yuzu/pull/3909), and Windows users will find that yuzu automatically detects their time zone. For those not on Windows (or want to spice up their life), you can manually change your system time via the "Custom RTC" option in the System settings. Previously, yuzu always assumed the user was located in the GMT+0 time zone.
+
+[bunnei](https://github.com/bunnei) also improved the [saving mechanism](https://github.com/yuzu-emu/yuzu/pull/3665). Most games save their data in each user profile, but 
+`Animal Crossing: New Horizons` does it via a “device” profile, so yuzu had to accommodate for that.
```

Comment:
Maybe explain a bit more the 'device saves'

Reply:
Not sure how to expand it, it's a save specific for your device.